tests/test_add.py

```diff
@@ -415,7 +415,7 @@ def test(self):
             ret = main(["add", self.FOO])
             self.assertEqual(0, ret)
 
-            foo_stage = os.path.abspath(self.FOO + Stage.STAGE_FILE_SUFFIX)
+            foo_stage = os.path.relpath(self.FOO + Stage.STAGE_FILE_SUFFIX)
```

Comment:
Not sure why this is needed.

Reply:
Do you mean relpath?
https://github.com/iterative/dvc/issues/1640#issuecomment-470214405

Reply:
@pared Ah, sorry. Missed that it is actually used down the line. Thanks!